interface/modules/custom_modules/oe-module-faxsms/src/Events/NotificationEventListener.php

```diff
@@ -137,7 +137,21 @@ public function onNotifyDocumentRenderOneTime(SendNotificationEvent $event): str
     }
 
     /**
-     * Send a token for universal onetime.
+     * Send a token for universal onetime/token.
+     * Example: Payment link.
+     * $data = [
+     *   'pid' => $e_pid,
+     *   'expiry_interval' => "P2D", // valid for 2 days.
+     *   'text_message' => "Please make a payment for your appointment.",
+     *   'html_message' => "",
+     *   'redirect_url' => $GLOBALS['web_root'] . "/portal/home.php?site=" . urlencode($_SESSION['site_id']) . "&landOn=MakePayment",
+     *   'actions' => [
+     *      'enforce_onetime_use' => true,
+     *      'enforce_auth_pin' => true,
+     *     ]
+     * ];
+     * // Dispatch the event. In this case, the onetime is created and emailed to the recipient.
+     * $GLOBALS["kernel"]->getEventDispatcher()->dispatch(new SendNotificationEvent($e_pid, $data, $to_where), SendNotificationEvent::SEND_NOTIFICATION_SERVICE_UNIVERSAL_ONETIME);
      *
```

Comment:
@juggernautsei 
Here is an example dispatched event where the listener will create the onetime and then email patient. Everything needed is passed in the event data. I also have javascript onetime create and buttons.

I'm working towards making the `onNotifyUniversalOneTime(SendNotificationEvent $event): string` universal to many use cases. I'll most likely move the listener to a more appropriate place besides in faxsmsemail module.

Next I'll be adding the onetime_actions to workflows in order to handle them. Some can be handled in `decodePortalOneTime($onetime_token, $redirect_token = null)` and `processOnetime($token, $redirect_token)`